packages/calcite-components/src/components/date-picker-month-header/date-picker-month-header.tsx

```diff
@@ -119,68 +149,130 @@ export class DatePickerMonthHeader {
-    const localizedYear = this.formatCalendarYear(activeDate.getFullYear());
-    const order = getOrder(unitOrder);
+    const order = getOrder(localeData.unitOrder);
     const reverse = order.indexOf("y") < order.indexOf("m");
-    const suffix = localeData.year?.suffix;
+
     return (
       <Fragment>
-        <a
-          aria-disabled={`${this.prevMonthDate.getMonth() === activeMonth}`}
-          aria-label={messages.prevMonth}
-          class={CSS.chevron}
-          href="#"
-          onClick={this.prevMonthClick}
-          onKeyDown={this.prevMonthKeydown}
-          role="button"
-          tabindex={this.prevMonthDate.getMonth() === activeMonth ? -1 : 0}
+        {this.position && (
+          <div class={{ [CSS.chevronContainer]: true }}>
+            {this.position === "start" && this.renderChevron("left")}
+          </div>
+        )}
+        <div
+          class={{
+            [CSS.monthYearContainer]: true,
+            [CSS.rangeCalendar]: !!this.position,
+          }}
         >
-          <calcite-icon flip-rtl icon={ICON.chevronLeft} scale={getIconScale(this.scale)} />
-        </a>
-        <div class={{ text: true, [CSS.textReverse]: reverse }}>
-          <Heading class={CSS.month} level={this.headingLevel}>
-            {localizedMonth}
-          </Heading>
-          <span class={CSS.yearWrap}>
-            <input
-              aria-label={messages.year}
-              class={{
-                year: true,
-                [CSS.yearSuffix]: !!suffix,
-              }}
-              inputmode="numeric"
-              maxlength="4"
-              minlength="1"
-              onChange={this.onYearChange}
-              onInput={this.onYearInput}
-              onKeyDown={this.onYearKey}
-              pattern="\d*"
-              ref={(el) => (this.yearInput = el)}
-              type="text"
-              value={localizedYear}
-            />
-            {suffix && <span class={CSS.suffix}>{suffix}</span>}
-          </span>
+          {this.renderMonthYearContainer(reverse)}
+        </div>
+        {!this.position && (
+          <div class={{ [CSS.chevronContainer]: true }}>{this.renderChevron("left")}</div>
+        )}
+        <div class={{ [CSS.chevronContainer]: true }}>
+          {this.position !== "start" && this.renderChevron("right")}
         </div>
-        <a
-          aria-disabled={`${this.nextMonthDate.getMonth() === activeMonth}`}
-          aria-label={messages.nextMonth}
-          class={CSS.chevron}
-          href="#"
-          onClick={this.nextMonthClick}
-          onKeyDown={this.nextMonthKeydown}
-          role="button"
-          tabindex={this.nextMonthDate.getMonth() === activeMonth ? -1 : 0}
-        >
-          <calcite-icon flip-rtl icon={ICON.chevronRight} scale={getIconScale(this.scale)} />
-        </a>
       </Fragment>
     );
   }
 
+  private renderMonthYearContainer(reverse: boolean): VNode {
+    return reverse ? (
+      <Fragment>
```

Comment:
Can we DRY this up?

```tsx
const content =  reverse ?  [/* ... */] : [/* ... */]; // or similar

return (
  <Fragment>{content}</Fragment>
);
```

Reply:
Didnt' knew we could use array in JSX. Very Cool & Clean.